sourabh/Html_assignment9.txt

```diff
@@ -0,0 +1,5 @@
+Q 9.1 If &lt;span&gt; and &lt;div&gt; are semantically neutral (that is, they do not confer any meaning to the
```

Comment:
Give me an example for what purpose you would use the "div" and "span" tag.
You can give your answer in reply section of this comment.

Reply:
this example i use in my program. 
this is example of div tag:- <div>This is my Div Tag</div>Check Block<br>
this is example of span tag:-<span>This is My Span Tag</span>Check In-line<br>